### PR TITLE
FIX: builds

### PR DESCRIPTION
**Pull request by eagerterrier**
input build phase

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
+"use strict";
```
Comment (andrewscfc) [inline dist/index.js]: should the build output be source controlled? Probably should be gitignored?

File: package.json [hunk 2]
```diff
@@ -12,7 +12,10 @@
     "test:ci": "yarn test:lint && yarn test:unit:ci",
     "test:lint": "eslint --ext .js,jsx,json ./scripts && stylelint 'scripts/**/*.js'",
     "test:unit": "yarn build && jest --verbose --coverage",
-    "test:unit:ci": "jest --verbose --coverage"
+    "test:unit:ci": "jest --verbose --coverage",
+    "build": "yarn build:cjs && yarn build:esm",
+    "build:cjs": "yarn exec babel src -d dist --ignore '**/*.test.js,**/*.test.jsx,**/*.stories.js,**/*.stories.jsx' --root-mode upward",
+    "build:esm": "BABEL_TYPE=esm yarn exec babel src -d esm --ignore '**/*.test.js,**/*.test.jsx,**/*.stories.js,**/*.stories.jsx' --root-mode upward"
```
Comment (andrewscfc) [inline package.json:19]: has this been taken from psammead?

Comment (eagerterrier) [reply package.json:19]: err. Maybe.

File: dist/index.js (hunk 1 above)
Comment (eagerterrier) [reply dist/index.js]: ooh - that's weird - was ignored on my working copy...

Comment (eagerterrier) [reply dist/index.js]: damn - ignore pattern was off - copied from psammead

File: package.json (hunk 2 above)
Comment (eagerterrier) [reply package.json:19]: anything odd in them? (I took at the many package things)

**Comment by eagerterrier** [comment]
closed in favour of https://github.com/bbc/web-vitals/pull/21

**Pull request closed without merging**